transforms/universal/web2parquet/dpk_web2parquet/config.py

```diff
@@ -0,0 +1,85 @@
+# (C) Copyright IBM Corp. 2024.
+# Licensed under the Apache License, Version 2.0 (the “License”);
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an “AS IS” BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+################################################################################
+
+import time
+import sys
+from argparse import ArgumentParser, Namespace
+from typing import Any
+
+import pyarrow as pa
+from data_processing.transform import TransformConfiguration
+from data_processing.utils import CLIArgumentProvider
+from data_processing.utils import get_logger
+from dpk_web2parquet.transform import Web2ParquetTransform
+
+short_name = "web2parquet"
+cli_prefix = f"{short_name}_"
+urls_cli_param = f"{cli_prefix}urls"
+depth_cli_param = f"{cli_prefix}depth"
+downloads_cli_param = f"{cli_prefix}downloads"
+folder_cli_param = f"{cli_prefix}folder"
+
+
+logger = get_logger(__name__,"DEBUG")
+    
+class Web2ParquetTransformConfiguration(TransformConfiguration):
+
+    """
+    Provides support for configuring and using the associated Transform class include
+    configuration with CLI args.
+    """
+
+    def __init__(self):
+        super().__init__(
+            name=short_name,
+            transform_class=Web2ParquetTransform
+        )
+
+    def add_input_params(self, parser: ArgumentParser) -> None:
+        """
+        Add Transform-specific arguments to the given  parser.
+        This will be included in a dictionary used to initialize the Web2ParquetTransform.
+        By convention a common prefix should be used for all transform-specific CLI args
+        (e.g, noop_, pii_, etc.)
+        """
+        parser.add_argument(f"--{depth_cli_param}", type=int, default=1,
+            help="maxumum depth relative to seed URL",
+        )
+        parser.add_argument(f"--{downloads_cli_param}", type=int, default=1,
+            help="maxumum number of downloaded URLs",
+        )
+        parser.add_argument(f"--{folder_cli_param}", type=str, default=None,
+            help="Folder wher to store downloaded files",
+        )
+        parser.add_argument(f"--{urls_cli_param}", type=str, default=None,
+            help="List of Seed URLs for the crawler",
+        )
```

Comment:
It looks good to me to start with a small set of the crawl parameters. We will be able to extend them in the future if there are requirements from the user.